spec/valkyrie/persistence/fedora/persister_spec.rb

```diff
@@ -3,170 +3,178 @@
 require 'valkyrie/specs/shared_specs'
 
 RSpec.describe Valkyrie::Persistence::Fedora::Persister do
-  let(:adapter) do
-    Valkyrie::Persistence::Fedora::MetadataAdapter.new(
-      connection: ::Ldp::Client.new("http://localhost:8988/rest"),
-      base_path: "test_fed",
-      schema: Valkyrie::Persistence::Fedora::PermissiveSchema.new(title: RDF::URI("http://bad.com/title"))
-    )
-  end
-  let(:persister) { adapter.persister }
-  let(:query_service) { adapter.query_service }
-  it_behaves_like "a Valkyrie::Persister"
-
-  context "when given an id containing a slash" do
-    before do
-      raise 'persister must be set with `let(:persister)`' unless defined? persister
-      class CustomResource < Valkyrie::Resource
-        include Valkyrie::Resource::AccessControls
-        attribute :title
-        attribute :author
-        attribute :member_ids
-        attribute :nested_resource
+  [4, 5].each do |fedora_version|
+    context "fedora #{fedora_version}" do
+      let(:fedora_version) { 4 }
```

Comment:
Is this overriding `fedora_version` on line 6 above?

Reply:
I don't think so, but just in case I changed it.